plugin-packages/rich-text/src/color-utils.ts

```
@@ -0,0 +1,55 @@
+   * 将颜色字符串转换为 RGBA
+   * @param color - 颜色字符串
+   * @param alpha - 透明度
+   * @returns - RGBA 颜色
+   */
+  static toRGBA (color: string, alpha: number = 1): spec.vec4 {
+    if (color.startsWith('#')) {
+      return this.hexToRGBA(color, alpha);
+    } else {
+      return this.hexToRGBA(this.colorNameToRGBA(color));
+    }
+  }
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)